Gem/Code/Source/Components/AWSMetricsSubmissionComponent.cpp

```diff
@@ -122,8 +137,23 @@ namespace MultiplayerSample
         MultiplayerStats& stats = AZ::Interface<IMultiplayer>::Get()->GetStats();
 
         AZStd::vector<AWSMetrics::MetricsAttribute> clientConnectCountMetricsAttributes;
-        clientConnectCountMetricsAttributes.emplace_back(AWSMetrics::MetricsAttribute("event_name", "client_connection_count"));
-        clientConnectCountMetricsAttributes.emplace_back(AWSMetrics::MetricsAttribute("client_connection_count", (int) stats.m_clientConnectionCount));
-        AWSMetrics::AWSMetricsRequestBus::Broadcast(&AWSMetrics::AWSMetricsRequests::SubmitMetrics, clientConnectCountMetricsAttributes, 0, AZStd::string("MultiplayerSample"), true);
+        clientConnectCountMetricsAttributes.emplace_back(AWSMetrics::MetricsAttribute(MetricsEventNameAttributeKey, ClientCountMetricsEvent));
+        clientConnectCountMetricsAttributes.emplace_back(AWSMetrics::MetricsAttribute(ClientCountMetricsEvent, (int) stats.m_clientConnectionCount));
+
+        bool result = false;
+        AWSMetrics::AWSMetricsRequestBus::BroadcastResult(result, &AWSMetrics::AWSMetricsRequests::SubmitMetrics, clientConnectCountMetricsAttributes, 0, MetricsEventSource, false);
+        if (!result)
+        {
+            AZ_TracePrintf(AWSMetricsSubmissionComponentName, "Failed to submit the %s metrics", ClientCountMetricsEvent);

```

Comment:
Consider making the log format consistent with the one below. IMO having the parameter at the end of the string makes for more searchable logs.

```suggestion
            AZ_TracePrintf(AWSMetricsSubmissionComponentName, "Failed to submit metrics: %s", ClientCountMetricsEvent);

```